libs/blocks/quiz/quiz.js

```diff
@@ -119,60 +125,29 @@ const App = ({
    */
   useEffect(() => {
     if (!nextQuizViewsExist && userSelection.length) {
+      const debugParam = initialUrlParams['debug-results'];
+      if (debugParam) {
+        const userSelectionString = JSON.stringify(userSelection);
+        const userSelectionStringEncoded = encodeURIComponent(userSelectionString);
+        navigator.clipboard.writeText(`${window.location.href}=${userSelectionStringEncoded}`);
```

Comment:
If you have other params in the URL, like martech=off, for example, appending the userSelection won't quite work. I think you'll need to do some URL parsing to form it correctly. Using the test URL, I wind up with a debugging URL like the following, which just goes back to the initial view of the quiz.

https://mwpw-125569-quiz-results-debugging--milo--fullcolorcoder.hlx.page/drafts/delta/app-recommender-test/?debug-results&martech=off=%5B%7B%22selectedQuestion%22%3A%7B%22questions%22%3A%22q%231%22%2C%22max-selections%22%3A%223%22%2C%22min-selections%22%3A%221%22%7D%2C%22selectedCards%22%3A%7B%22photo%22%3Atrue%7D%7D%2C%7B%22selectedQuestion%22%3A%7B%22questions%22%3A%22q%232%22%2C%22max-selections%22%3A%221%22%2C%22min-selections%22%3A%221%22%7D%2C%22selectedCards%22%3A%7B%22template%22%3Atrue%7D%7D%2C%7B%22selectedQuestion%22%3A%7B%22questions%22%3A%22q%234%22%2C%22max-selections%22%3A%221%22%2C%22min-selections%22%3A%221%22%7D%2C%22selectedCards%22%3A%7B%22ste%22%3Atrue%7D%7D%5D

Tweaking it so the debug-results param gets the value does the trick. 

https://mwpw-125569-quiz-results-debugging--milo--fullcolorcoder.hlx.page/drafts/delta/app-recommender-test/?debug-results=%5B%7B%22selectedQuestion%22%3A%7B%22questions%22%3A%22q%231%22%2C%22max-selections%22%3A%223%22%2C%22min-selections%22%3A%221%22%7D%2C%22selectedCards%22%3A%7B%22photo%22%3Atrue%7D%7D%2C%7B%22selectedQuestion%22%3A%7B%22questions%22%3A%22q%232%22%2C%22max-selections%22%3A%221%22%2C%22min-selections%22%3A%221%22%7D%2C%22selectedCards%22%3A%7B%22template%22%3Atrue%7D%7D%2C%7B%22selectedQuestion%22%3A%7B%22questions%22%3A%22q%234%22%2C%22max-selections%22%3A%221%22%2C%22min-selections%22%3A%221%22%7D%2C%22selectedCards%22%3A%7B%22ste%22%3Atrue%7D%7D%5D

I also can't quite decide how I feel about the clipboard. Part of me would rather just have it sent to the console. I definitely accept the argument that the clipboard is convenient, but I like a paper trail. Since we're redirecting, the clipboard permissions popup often just dismisses itself on the way and I don't get the value.

Reply:
Thanks for the review! I cleaned up the URL, and added support for console to have the paper trail. You'll need to check "Preserve logs" to not lose it though when hitting submit.